demos/button.html

```diff
@@ -66,345 +80,116 @@
-          </button>
-          <button class="mdc-button mdc-button--primary">
-            Default with Primary
-          </button>
-          <button class="mdc-button mdc-button--raised mdc-button--primary">
-            Raised with Primary
-          </button>
-          <button class="mdc-button mdc-button--accent">
-            Default with Accent
-          </button>
-          <button class="mdc-button mdc-button--raised mdc-button--accent">
-            Raised with Accent
-          </button>
-          <div class="mdc-button mdc-button--raised" tabindex="0" role="button">
-            Div
-          </div>
-        </fieldset>
-
-        <fieldset>
-          <legend class="mdc-typography--title">Buttons CSS Only</legend>
-          <button class="mdc-button" data-demo-no-js>
-            Default
-          </button>
-          <button class="mdc-button mdc-button--raised" data-demo-no-js>
-            Raised
-          </button>
-          <button class="mdc-button mdc-button--dense" data-demo-no-js>
-            Dense Default
-          </button>
-          <button class="mdc-button mdc-button--raised mdc-button--dense" data-demo-no-js>
-            Dense Raised
-          </button>
-          <button class="mdc-button mdc-button--compact" data-demo-no-js>
-            Compact
-          </button>
-          <button class="mdc-button mdc-button--raised mdc-button--compact" data-demo-no-js>
-            Compact Raised
-          </button>
-          <button class="mdc-button mdc-button--primary" data-demo-no-js>
-            Default with Primary
-          </button>
-          <button class="mdc-button mdc-button--raised mdc-button--primary" data-demo-no-js>
-            Raised with Primary
-          </button>
-          <button class="mdc-button mdc-button--accent" data-demo-no-js>
-            Default with Accent
-          </button>
-          <button class="mdc-button mdc-button--raised mdc-button--accent" data-demo-no-js>
-            Raised with Accent
-          </button>
-          <div class="mdc-button mdc-button--raised" tabindex="0" role="button" data-demo-no-js>
-            Div
+      <section class="demo-wrapper">
+        <div class="mdc-form-field">
+          <div class="mdc-checkbox">
+            <input type="checkbox" class="mdc-checkbox__native-control" id="toggle-dark" aria-labelledby="toggle-dark-label" />
+            <div class="mdc-checkbox__background">
+              <svg class="mdc-checkbox__checkmark" viewBox="0 0 24 24">
+                <path class="mdc-checkbox__checkmark__path" fill="none" stroke="white" d="M1.73,12.91 8.1,19.28 22.79,4.59" />
+              </svg>
+              <div class="mdc-checkbox__mixedmark"></div>
+            </div>
           </div>
-        </fieldset>
+          <label for="toggle-dark" id="toggle-dark-label">Dark Theme</label>
+        </div>
 
         <fieldset>
-          <legend class="mdc-typography--title">Links with Button Style</legend>
```

Comment:
It was because I modified the file from the original BREAKING CHANGE pr with string substitution, I don't feel this make much of a difference though?